draft-ietf-grow-nrtm-v4.xml

```diff
@@ -554,24 +557,22 @@
       </t>
       <t>
 <artwork><![CDATA[
-{
+␞{
   "nrtm_version": 4,
   "type": "snapshot",
   "source": "EXAMPLE",
   "session_id": "ca128382-78d9-41d1-8927-1ecef15275be",
-  "version": 3,
-  "objects": [
-    "route: 192.0.2.0/24\norigin: AS65530\nsource: EXAMPLE",
-    "route: 2001:db8::/32\norigin: AS65530\nsource: EXAMPLE"
-  ]
+  "version": 3
 }
+␞"route: 192.0.2.0/24\norigin: AS65530\nsource: EXAMPLE"
```

Comment:
You have defined the snapshot file to use JSON text sequence format, the object string need not be JSON itself, but is it easier to parse each line as a valid JSON string? For example:
```{"object": "route: 192.0.2.0/24\norigin: AS65530\nsource: EXAMPLE"}```
This is also more consistent with the delta file, where each change is a valid JSON string.

Reply:
Yes, this was an oversight.